presto-hive/src/main/java/io/prestosql/plugin/hive/security/LegacyAccessControl.java

```diff
@@ -269,6 +270,12 @@ public void checkCanShowRoleGrants(ConnectorSecurityContext context, String cata
     {
     }
 
+    @Override
+    public void checkCanExecuteProcedure(ConnectorSecurityContext context, SchemaProcedureName procedure)
+    {
+        //TODO implement
```

Comment:
The todo isn't needed. The `LegacyAccessControl` is designed to be backwards compatible with the original Presto Hive security system, and therefore we should I don't think we should be implementing this here.

Reply:
From backwards compaitbility perspective... procedures didn't really exist until recently. We opened some problem by adding them and we should somehow fix it.

Reply:
I disagree.  The legacy security system should be removed.  I believe the first step it to change the default from legacy to something else... likely sql standard.